src/exampleWorker.js

```diff
@@ -0,0 +1,229 @@
+/* eslint-env browser */
+
+'use strict';
+
+self.importScripts('../dist/ohm.js', 'utils.js');
+
+// Web Worker that generates examples.
+// communicates with 'exampleGenerationUI.js'
+
+(function(root, initModule) {
+  if (typeof exports === 'object') {
+    module.exports = initModule;
+  } else {
+    initModule(root.ohm, root.utils);
+  }
+})(this, function(ohm, utils) {
+  postMessage('WORKER STARTED');
+  var grammar;
+
+  var examplePieces = {};
+  var generator;
+
+  var overrides = {
+    ident: function(g, ex, syn, args) {
+      return {needHelp: true};
+    },
+    ctor: function(g, ex, syn, args) {
+      return {needHelp: true};
+    },
+    letter: function(g, ex, syn, args) {
+      if (Math.floor(Math.random() * 2) === 0) {
```

Comment:
Since you use `Math.floor(Math.random() ...)` so often, maybe it's worth making a `randInt(max)` or `randInt(min, max)` function? It would make the code slightly easier to read.